Tasks/PackerBuildV1/src/taskParameters.ts

```diff
@@ -83,7 +83,7 @@ export default class TaskParameters {
 
                 this.deployScriptArguments = tl.getInput(constants.DeployScriptArgumentsInputName, false);
 
-                this.graphCredentials = this._getAzureADGraphCredentials(this.serviceEndpoint);
+                this.graphCredentials = this.getGraphCredentials(this.serviceEndpoint);
```

Comment:
I think the PackerV1 and PackerV0 tasks versions are not updated. Please do so.

Reply:
It's updated